start.py

```diff
@@ -58,7 +59,7 @@ def main(host: str, port: int, reload: bool, workers: int | None=None):
         print(str(e))
 
     # Start the application
-    uvicorn.run("app.main:app", host=host, port=port, reload=reload, workers=workers)
+    uvicorn.run("app.main:app", host=host, port=port, reload=reload, workers=workers, log_config=None)
```

Comment:
If we pass a config in through here it messes things up, because it sets up the default logger. We want to disable the default logger (uvicorn, uvicorn.access, etc.) because those would double log everything.